arviz/plots/autocorrplot.py

```diff
@@ -23,37 +23,38 @@ def plot_autocorr(
     backend_kwargs=None,
     show=None,
 ):
-    """Bar plot of the autocorrelation function for a sequence of data.
+    """Bar plot of the autocorrelation function (ACF) for a sequence of data.
 
-    Useful in particular for posteriors from MCMC samples which may display correlation.
+    The ACF plots are helpful as a convergence diagnostic for posteriors from MCMC 
+    samples which may display correlation. The ACF is efficiently computed using a 
+    Fast Fourier Transform for every lag for the input array.
```

Comment:
```suggestion
    samples which display autocorrelation.
```

I would add the "The ACF is efficiently computed using a Fast Fourier Transform for every lag for the input array." part in the notes, ideally also with a reference on using fft for autocov computation (or maybe moving this to the autocov or autocorr functions instead of this one).

For the note on their use, I think this is a bit more clear? I am not aware of using autocorr plot for diagnosing posteriors with correlation between parameters

Reply:
I decided to explain it here because I didn't consider it specific enough to have a note, but ok. and MMM I don't think autocor is for correlation with between parameters